_includes/pbc/all/install-features/202407.0/install-the-product-management-ai-feature.md

```diff
@@ -0,0 +1,254 @@
+     */
+    protected function getProductConcreteEditFormExpanderPlugins(): array
+    {
+        return [
+            ...
+            new ImageAltTextProductConcreteEditFormExpanderPlugin(),
+        ];
+    }
+
+    /**
+     * @return array<\Spryker\Zed\ProductManagementExtension\Dependency\Plugin\ProductAbstractFormExpanderPluginInterface>
+     */
+    protected function getProductAbstractFormExpanderPlugins(): array
+    {
+        return [
+            ...
+            new ProductCategoryAbstractFormExpanderPlugin(),
+            new ImageAltTextProductAbstractFormExpanderPlugin(),
+        ];
+    }
+
+    /**
+     * @return array<\Spryker\Zed\ProductManagementExtension\Dependency\Plugin\ProductConcreteFormExpanderPluginInterface>
+     */
+    protected function getProductConcreteFormExpanderPlugins(): array
+    {
+        return [
+            ...
+            new ImageAltTextProductConcreteFormExpanderPlugin(),
+        ];
+    }
+```
+
+**src/Pyz/Zed/ProductManagement/Presentation/Add/index.twig**
+
+```twig
+{% raw %}
+{% extends '@Spryker:ProductManagement/Add/index.twig' %}
+
+{% block content %}
+    {{ parent() }}
+
+    {% include '@SprykerEco:ProductManagementAi/_partials/product-management-ai-modals.twig' %}
+{% endblock %}
+{% endraw %}
+```
+
+**src/Pyz/Zed/ProductManagement/Presentation/Edit/index.twig**
+
+```twig
+{% raw %}
+{% extends '@Spryker:ProductManagement/Edit/index.twig' %}
+
+{% block content %}
+    {{ parent() }}
+
+    {% include '@SprykerEco:ProductManagementAi/_partials/product-management-ai-modals.twig' %}
+{% endblock %}
+{% endraw %}
+```
+
+**src/Pyz/Zed/ProductManagement/Presentation/Product/_partials/general-tab.twig**
+
+```twig
+{% raw %}
+{% extends '@Spryker:ProductManagement/Product/_partials/general-tab.twig' %}
+
+{% block descriptionField %}
+    {% embed '@Gui/Partials/localized-ibox.twig' with {'collapsed': (not loop.first), 'localeName': locale.localeName} %}
+        {% block content %}
+            {% for input in form[formKey] %}
+                {% embed '@SprykerEco:ProductManagementAi/_partials/ai-translation-trigger.twig' %}
+                    {% block input %}
+                        <div class="form-group {% if input.vars.errors|length %}has-error{% endif %}">
+                            {{ form_label(input) }}
+                            {{ form_widget(input, {'attr': {'class': 'name-translation js-infomational-field'}}) }}
+                            {{ form_errors(input) }}
+                        </div>
+                    {% endblock %}
+                {% endembed %}
+            {% endfor %}
+        {% endblock %}
+    {% endembed %}
+{% endblock %}
+
+{% block description %}
+{{ parent() }}
+
+    {% include '@SprykerEco:ProductManagementAi/_partials/category-modal-trigger.twig' %}
+{% endblock %}
+{% endraw %}
+```
+
+To apply the changes, run the following command:
```

Comment:
Apply changes sounds not correct, please describe properly what this command does